spec/cdn_source_spec.rb

```diff
@@ -112,7 +112,7 @@ def cleanup
         should.raise Informative do
           @source.specification('BeaconKit', '1.0.0')
         end.message.
-          should.== "CDN: #{@source.name} Relative path couldn\'t be downloaded: Specs/2/0/9/BeaconKit/1.0.0/BeaconKit.podspec.json Response: 500"
+          should.== "CDN: #{@source.name} Relative path couldn\'t be downloaded: #{@source.url}Specs/2/0/9/BeaconKit/1.0.0/BeaconKit.podspec.json Response: 500"
```

Comment:
its generally advisable not to use the actual code for assertions as you might end up with a false positive. Recommend putting the actual string here.

Reply:
Also I see that it does it in the beginning so maybe its fine.

Reply:
Makes sense. I was just following the example of `#{@source.name}`.